exersices/Ex01/letter_counts.py

```diff
@@ -0,0 +1,24 @@
+def letter_freq(txt):
+    liten = txt.lower()
+    alpha = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j',
```

Comment:
Nice idea.
However, are you sure you can list all the existing characters? `@ # $ % ^ & ( )` are still missing, and this list can even grow more.

The idea you are using is to count every character you know in the string. However, do you think you can count any character that appears in the string?

Instead of making a list of characters (which may not enough), you can go through each character in the string, mark it as counted 1, then increase by 1 when you see it again. (You may want to see the suggested solution for that).